src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinstallment.phtml

```diff
@@ -14,11 +22,18 @@ $backends = $this->getInstallmentBackends();
                 <div class="logo-<?=$backend->_bankcode;?>"></div>
             </div>
             <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$backend->_bankname;?></span><br/>
+                <span class="title"><?=$installmentText;?></span><br/>
```

Comment:
formatting? `<?= $backend->_bankname; ?>`

the same with `installmentText`

Reply:
Why? I prefer this way since forever